File: 01_Language/01_Functions/php/lcfirst.php
<?php

$foo = 'HelloWorld';
$foo = lcfirst($foo);             // helloWorld
var_dump($foo);

$bar = 'HELLO WORLD!';
$bar = lcfirst($bar);             // hELLO WORLD!
$bar = lcfirst(strtoupper($bar)); // hELLO WORLD!
var_dump($bar);

var_dump(lcfirst(" Abc"));
var_dump(lcfirst(null));

File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/Lcfirst.java
package com.onevgo.functions;

import org.apache.commons.lang3.StringUtils;

public class Lcfirst {
    public static String lcfirst(String str) {
        return str == null ? "" : StringUtils.uncapitalize(str);
    }

    public static void main(String[] args) {
        String foo = "HelloWorld";
        System.out.println(lcfirst(foo));

        String bar = "HELLO WORLD!";
        System.out.println(lcfirst(bar));

        System.out.println(lcfirst(" Abc"));
        System.out.println(lcfirst(null));
    }
}
